pkg/dss/auth/auth_test.go

```diff
@@ -109,18 +109,19 @@ func TestRSAAuthInterceptor(t *testing.T) {
 			Key: &key.PublicKey,
 		},
 		KeyRefreshTimeout: 1 * time.Millisecond,
+		RequiredAudiences: []string{""},
 	})
 
 	require.NoError(t, err)
 
-	for _, test := range authTests {
+	for i, test := range authTests {
 		_, err := a.AuthInterceptor(test.ctx, nil, &grpc.UnaryServerInfo{},
```

Comment:
Why not do `t.Run(strconv.Itoa(i), t.Run(...)`. With that, you can avoid manually printing the index in the error messages.